src/app/search/page.tsx

```diff
@@ -13,118 +15,124 @@ import {
   PaginationNext,
   PaginationPrevious,
 } from '@/components/ui/pagination';
-import { convertStringToDate } from '@/lib/dateTime';
+
+import { useSearchProjects } from '@/hooks/queries/useProjectService';
+import { useSearchStore } from '@/stores/searchStore';
+
+import { convertTimestampToDate } from '@/lib/dateTime';
+import { cn } from '@/lib/utils';
+
+const ITEMS_PER_PAGE = 8; // 한 페이지에 나올 아이템의 최대 개수
+const DEFAULT_ITEMS_TOTAL_COUNT = 1; // 데이터를 받아오기 전, 아이템 총 개수 기본값 / 한 페이지만 떠도 되니까 1로 둠
 
 export default function SearchPage() {
-  const [currentPage, setCurrentPage] = useState(1);
+  // 검색 조건
+  const searchCondition = useSearchStore((state) => state.searchCondition);
+
+  // 현재 검색 페이지
+  const [currentPage, setCurrentPage] = useState(0); // 0부터 시작, 0이 1페이지
+
+  // 조건에 따른 검색 결과 (api)
+  const { data, isLoading, isError } = useSearchProjects({
+    searchType: searchCondition.type,
+    searchWord: searchCondition.keyword,
+    categories: searchCondition.categories,
+    pageNo: currentPage,
+    pageSize: ITEMS_PER_PAGE, // 한 화면에 8개 고정
+  });
+
+  // 검색 결과 데이터 프로세싱
+  const searchProjects =
+    data?.data.contents.map((content) => {
+      return {
+        projectId: content.projectId,
+        projectname: content.projectName,
```

Comment:
```suggestion
        projectName: content.projectName,
```
근데 계속 projectname으로 쓰였네요,,, 통일해주는게 좋을 것 같습니다!

Reply:
✅ f65b25eb281f4982beec42b0a9b71164276d5824